lib/auth/oidc.go

```diff
@@ -369,8 +369,15 @@ func checkEmailVerifiedClaim(claims jose.Claims) error {
 	unverifiedErr := trace.AccessDenied("email not verified by OIDC provider")
 
 	emailVerified, hasEmailVerifiedClaim, _ := claims.StringClaim(claimName)
-	if hasEmailVerifiedClaim && emailVerified == "false" {
-		return unverifiedErr
+	if hasEmailVerifiedClaim {
+		if emailVerified == "false" {
```

Comment:
Should we use `strings.ToLower` here and below just in case?

Reply:
We decided not to. Want to be as strict as possible since we're already out of spec here.